### ISSUE TITLE
Add a CLI option to show staged changes

### ISSUE THREAD

**Issue by teunvink** [raulgomis/semversioner#3]
An option to see which changes (and of what type they are) are currently staged for the next release would be nice. A "prediction" what the next version number be based on the currently changed changes would be a nice addition.

PS: thanks for this tool, this was just what I was looking for for a number of my projects!

**Comment by raulgomis** [comment]
Hi @teunvink ,

Glad you find this tool useful for your projects. The feature request makes total sense to me, it's a great suggestion that other users could benefit. 

I gave some thought to this and I was thinking about adding a new command called `status` that shows the current version, the next version and the list of unreleased changes. It also shows a text message indicating usage recommendations to make it easier for users to know next steps. It is similar to how `git status` works in order to make it really simple for majority of users.

Example: 

In this example, the last released version was 0.9.0, and there is one unreleased change (major) with description "This is the description":

```
$ semversioner status
Version: 0.9.0
Next version: 1.0.0
Unreleased changes:
        major:  This is the description
(use "semversioner release" to release the next version)
```

Would that work for your particular case?

**Comment by teunvink** [comment]
Sounds like a great solution, thanks for the quick action!
